docs/reference/api-docs.asciidoc

```diff
@@ -56,6 +56,7 @@ Agent is the Schema for the Agents API.
 | *`metadata`* __link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#objectmeta-v1-meta[$$ObjectMeta$$]__ | Refer to Kubernetes API documentation for fields of `metadata`.
 
 | *`spec`* __xref:{anchor_prefix}-github-com-elastic-cloud-on-k8s-v2-pkg-apis-agent-v1alpha1-agentspec[$$AgentSpec$$]__ | 
+| *`status`* __xref:{anchor_prefix}-github-com-elastic-cloud-on-k8s-v2-pkg-apis-agent-v1alpha1-agentstatus[$$AgentStatus$$]__ | 
```

Comment:
Not sure where does this change come from... any idea @thbkrkr?

Reply:
```

INFO:build_docs:
--
  | INFO:build_docs:
  | INFO:build_docs:asciidoctor: WARNING: invalid reference: k8s-api-github-com-elastic-cloud-on-k8s-v2-pkg-apis-agent-v1alpha1-agentstatus
  | INFO:build_docs:
  | INFO:build_docs:Child exited with 255 at lib/ES/Util.pm line 666

```

😞

Reply:
Regression might have been introduced in https://github.com/elastic/crd-ref-docs/pull/25